src/Sonarr.Api.V3/Profiles/Quality/QualityCutoffValidator.cs

```diff
@@ -15,30 +15,17 @@ public static class QualityCutoffValidator
 
     public class ValidCutoffValidator<T> : PropertyValidator
     {
-        public ValidCutoffValidator()
-            : base("Cutoff must be an allowed quality or group")
-        {
-        }
+        protected override string GetDefaultMessageTemplate() => "Cutoff must be an allowed quality or group";
 
         protected override bool IsValid(PropertyValidatorContext context)
         {
             var cutoff = (int)context.PropertyValue;
             dynamic instance = context.ParentContext.InstanceToValidate;
             var items = instance.Items as IList<QualityProfileQualityItemResource>;
 
-            var cutoffItem = items.SingleOrDefault(i => (i.Quality == null && i.Id == cutoff) || i.Quality?.Id == cutoff);
-
-            if (cutoffItem == null)
-            {
-                return false;
-            }
-
-            if (!cutoffItem.Allowed)
-            {
-                return false;
-            }
+            var cutoffItem = items?.SingleOrDefault(i => (i.Quality == null && i.Id == cutoff) || i.Quality?.Id == cutoff);
```

Comment:
The same behavior except the situation when items is null.
Previously, this would have been an null ref exception, but now we just return false.
So the same question here, should we return true instead?

Reply:
I think false is fine if the expected list is null.